src/main/java/mate/academy/hibernate/relations/Main.java

```diff
@@ -15,18 +24,21 @@ public static void main(String[] args) {
         // use this session factory when you will initialize service instances
         SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
 
+        CountryDao countryDao = new CountryDaoImpl(sessionFactory);
         Country usa = new Country("USA");
-        CountryService countryService = null; // TODO: initialize this instance
+        CountryService countryService = new CountryServiceImpl(countryDao);
         countryService.add(usa);
 
+        ActorDao actorDao = new ActorDaoImpl(sessionFactory);
         Actor vinDiesel = new Actor("Vin Diesel");
         vinDiesel.setCountry(usa);
-        ActorService actorService = null; // TODO: initialize this instance
+        ActorService actorService = new ActorServiceImpl(actorDao);
```

Comment:
```suggestion
        ActorService actorService = new ActorServiceImpl(new ActorDaoImpl(sessionFactory));
```
you can omit declaring variable for ActorDao